dbms/src/Storages/StorageReplicatedMergeTree.h

```diff
@@ -212,6 +212,8 @@ class StorageReplicatedMergeTree : public ext::shared_ptr_helper<StorageReplicat
     using LogEntry = ReplicatedMergeTreeLogEntry;
     using LogEntryPtr = LogEntry::Ptr;
 
+    using PartitionIdToMaxBlock = std::unordered_map<String, Int64>;
```

Comment:
This typedif is not used anywhere in the interface of StorageReplicatedMergeTree, but used in MergeTreeDataSelectExecutor. I think it is better to move the typedef there.